packages/firebase_ui_storage/example/windows/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+# Project-level configuration.
+cmake_minimum_required(VERSION 3.14)
+project(firebase_ui_example LANGUAGES CXX)
+
+# The name of the executable created for the application. Change this to change
+# the on-disk name of your application.
+set(BINARY_NAME "firebase_ui_example")
+
+# Explicitly opt in to modern CMake behaviors to avoid warnings with recent
```

Comment:
Same for windows folder

Reply:
`flutterfire_desktop` will work on windows too :)